tests/core/test_lightning_module.py

```diff
@@ -399,3 +400,11 @@ def configure_gradient_clipping(self, optimizer, optimizer_idx, gradient_clip_va
         match=r"gradient_clip_algorithm='norm'\)` and have passed `clip_gradients\(gradient_clip_algorithm='foo'",
     ):
         trainer.fit(model)
+
+
+@RunIf(skip_windows=True, min_torch="1.10")
```

Comment:
qq:
why skip on windows and min torch?

Reply:
Since it's a small test, I suggest removing the skip.

Even if it's not a bug under those circumstances, it's a nice check to have

```suggestion
```